### PR TITLE
Add finetuning loss function tutorial and link it to finetuning tutorial

### PR DESCRIPTION
**Pull request by marcopeix**
New tutorial on setting a loss function for fine-tuning
- Comparison between each loss function and the default on how it improves its associated error metric
- Basic usage for setting the loss function
- The existing fine-tuning tutorial links to this tutorial on using different loss functions
- Added sklearn as a dev requirement to compute the different metrics. I thought of using neuralforecast to compute the metrics, but I felt it was overkill, but I could change that so we stay in the Nixtlaverse

### CODE REVIEW

File: nbs/docs/tutorials/10_loss_function_finetuning.ipynb [hunk 1]
```diff
@@ -0,0 +1,602 @@
+{
```
Comment (AzulGarza) [inline nbs/docs/tutorials/10_loss_function_finetuning.ipynb:1]: <p>it might be nice to have the equation of the metrics, <a href="https://github.com/Nixtla/utilsforecast/blob/main/nbs/losses.ipynb" target="_blank">here</a> you can find them.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/239/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Comment (AzulGarza) [inline nbs/docs/tutorials/10_loss_function_finetuning.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #2.    from sklearn.metrics import mean_absolute_error, mean_squared_error, root_mean_squared_error, mean_absolute_percentage_error
</pre><p>could we use these metrics from the <a href="https://github.com/Nixtla/utilsforecast" target="_blank">utilsforecast</a> library? :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/239/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (AzulGarza) [inline nbs/docs/tutorials/10_loss_function_finetuning.ipynb:1]: <p>there is a typo here: <code>exmaple</code> -&gt; <code>example</code>. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/nixtla/pull/239/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Comment (marcopeix) [reply nbs/docs/tutorials/10_loss_function_finetuning.ipynb:1]: Used utilsforecast and removed scikit-learn from the dev dependencies!

Comment (marcopeix) [reply nbs/docs/tutorials/10_loss_function_finetuning.ipynb:1]: Fixed the typo!

**Comment by marcopeix** [comment]
PR reviewed as per the comments! Using utilsforecast for model evaluation, fixed the typo and added the equations for each loss function.

**Comment by AzulGarza** [comment]
awesome @marcopeix! could you sign the license/cla? 

> [CLA assistant check](https://cla-assistant.io/Nixtla/nixtla?pullRequest=239) Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Nixtla/nixtla?pullRequest=239) before we can accept your contribution.
> 
> **Marco Peixeiro** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).
> You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Nixtla/nixtla?pullRequest=239) it.

also, a small change: could you change the tutorial's name to `*11*_losss_function...`? yesterday a new tutorial was added with *10_...* 🙈

**Comment by marcopeix** [comment]
Renamed the notebook :)

**Comment by jmoralez** [comment]
@AzulGarza wdyt about wrapping the models' performance comment in a `<details>` tag so that it's collapsed by default (like [here](https://github.com/Nixtla/neuralforecast/issues/902#issuecomment-1965620592))? Otherwise it's very difficult to follow a conversation.

**Comment by AzulGarza** [comment]
> @AzulGarza wdyt about wrapping the models' performance comment in a `<details>` tag so that it's collapsed by default (like [here](https://github.com/Nixtla/neuralforecast/issues/902#issuecomment-1965620592))? Otherwise it's very difficult to follow a conversation.

@jmoralez love it! fixed in #247.
